.travis.yml

```diff
@@ -1,12 +1,40 @@
-# Build config for travis-ci.org
 language: node_js
 node_js:
   - 5.5
+  - 6
+
+addons:
+  mariadb: '5.5'
+
+services:
+  - mariadb
```

Comment:
https://docs.travis-ci.com/user/database-setup/#Starting-Services

"If you install a service in the addons: section, such as MariaDB, you do not need to add it to the services: section as well."